### DISCUSSION TITLE
FluxD Lora models from civitai

### DISCUSSION THREAD

**Discussion by pes1972** [lllyasviel/stable-diffusion-webui-forge#1628] [Q&A]
No matter which FluxD Lora model from civitai i try they all just becomes any random woman when generating them in Forge. Anyone have any hints or could post their settings used in forge to recreate Flux lora models.

**Comment by vonespy** [comment]
Post a screenshot of your settings and more system info (nvidia card and memory, system memory, etc.) and someone will be more likely to help.
